bokehjs/src/coffee/core/util/types.ts

```diff
@@ -0,0 +1,30 @@
+const toString = Object.prototype.toString
+
+export function isBoolean(obj: any): obj is boolean {
+  return obj === true || obj === false || toString.call(obj) === '[object Boolean]'
+}
+
+export function isNumber(obj: any): obj is number {
+  return toString.call(obj) === "[object Number]"
+}
+
+export function isString(obj: any): obj is string {
+  return toString.call(obj) === "[object String]"
+}
+
+export function isStrictNaN(obj: any): obj is number {
+  return isNumber(obj) && obj !== +obj
+}
```

Comment:
Ok, but then please make the issue now to investigate later (and reference this PR)